.buildkite/build-static-release.sh

```diff
@@ -16,7 +16,11 @@ case "$platform" in
   linux-aarch64)
     CONFIG_OPTS="--config=release-${platform}"
     ;;
-  darwin-x86_64|darwin-arm64)
+  darwin-x86_64)
+    CONFIG_OPTS="--config=release-mac"
+    command -v autoconf >/dev/null 2>&1 || brew install autoconf
+    ;;
+  darwin-arm64)
```

Comment:
What's the difference between these two cases now? the darwin-arm64 case has the same logic right?

Also, is the darwin-arm64 case untested, because we don't have any macOS arm64 machines in buildkite?

That means that we still have the problem that

```
./bazel build --config=release-mac-arm64 //main:sorbet
```

doesn't produce working binaries when we run it on our laptops, doesn't it?

Reply:
Ah, I see—it's because the release-common / shared-libs thing changed too.